benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1696,6 +1696,11 @@ void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, si
             frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[1], 3, &colorValues);
             break;
         }
+        case FullscreenQuadsType::FULLSCREEN_QUADS_TYPE_TEXTURE:
+            break;
+        default: {
+            PPX_ASSERT_MSG(true, "unsupported FullscreenQuadsType: " << static_cast<int>(pFullscreenQuadsType->GetValue()));
```

Comment:
Could you add a `break` in `default`? and I don't think we need `{}` there either.
The break wouldn't matter now, but if we ever added a case in the end or move the default to top we might forget to add break then

Reply:
OK, done!